system/services/errors/ErrorLogService.cfc

```diff
@@ -54,7 +54,10 @@ component displayName="Error Log Service" {
 		var errors = [];
 
 		for( var file in files ) {
-			errors.append( { date=file.dateLastModified, filename=file.name } );
+			var errorMessage   = REMatchNoCase( "<\s*title[^>]*>(.*?)<\s*\/\s*title>", FileRead( _getLogDirectory() & "/" & file.name ) );
+			var messageContent = ReplaceList( errorMessage[1] , "<title>,</title>", "#chr(10)# ,#chr(10)#" );
```

Comment:
We ought to be a little careful here. What if, for whatever reason, `REMatchNoCase()` returns an empty array here? 

Perhaps: 
```
var messageContent = errorMessage.len() ? ReplaceList( errorMessage[1] , "<title>,</title>", "#chr(10)# ,#chr(10)#" ) : "";
```